app/serializable/serializable_submission.rb

```diff
@@ -26,9 +27,19 @@ class SerializableSubmission < JSONAPI::Serializable::Resource
     Hash[submission.sheet_names.map { |sheet_name| [sheet_name, errors_for(sheet_name)] }]
   end
 
+  attribute :invoice_total_value do
```

Comment:
rather than this (and the order) logic living here in the serialiser, perhaps it is better to have it in the `Submission` model so it's available in more places and also easier/simpler to test.

Reply:
Another issue; having this exposed here introduces a potentially costly N+1 here: https://github.com/dxw/DataSubmissionService/blob/89cc510fefc9a972448169b62b692ccb3c6121d1/app/controllers/tasks_controller.rb#L5 – the API will run two `SUM` queries for every task with a submission. V1::TasksController#index is already [our most costly request](https://www.skylight.io/app/applications/FBaSsvnUNghr/recent/6h/endpoints) and this is likely to make it significantly worse. Is there something we can do to mitigate this?